src/Components/FormInput/formInput.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styles from './formInput.module.css'
+
+export const FormInput = ({ labelText, type, value, onChange }) => {
+  return (
+    <div className={styles.formDiv}>
+      <label className={styles.formLabel}>{labelText}</label>
+      <input className={styles.formInput} type={type} value={value} onChange={onChange} />
```

Comment:
Would it be better to use `defaultValue`  instead of `value`? Otherwise, you can use this component only as a controlled component but there is no state defined in this component.